src/macsniff.cpp

```diff
@@ -38,7 +38,8 @@ bool mac_add(uint8_t *paddr, int8_t rssi, bool sniff_type) {
 	uint16_t hashedmac;
 
     // Only last 3 MAC Address bytes are used bay MAC Address Anonymization
-    addr2int =  ( (uint32_t)paddr[3] ) | ( (uint32_t)paddr[4] << 8 ) | ( (uint32_t)paddr[5] << 16 );
+    // but since it's uint32 we take 4 bytes to avoid 1st value to be 0
```

Comment:
This is why i OR'ed the salt, noch ADD'ed it

addr2int += (uint32_t) salt;

(was: addr2int |= (uint32_t) salt;)

Reply:
yes but in this case or is giving less "possibly" of change than a classic addition 
and now if a classic addition of 16bit value into a 32bits one if overflowing the 16bits it not an issue because we take the last 5 char of hex value buffer so like a 24bits value (ie 12345)